src/components/Article/Articleform.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable react/destructuring-assignment */
+import React, { Component } from 'react';
+import axios from 'axios';
+
+import './Articles.scss';
+
+class ArticleForm extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
```

Comment:
If you are not using the local state just remove it so that you do not need to suppress the linter.